audits/UsmannKhan_fixReview.md

```diff
@@ -0,0 +1,362 @@
+# papr Fix Review
+papr engaged Usmann Khan to conduct a fix review of their changes made in response to findings from their [Code4rena contest](https://code4rena.com/contests/2022-12-papr-contest). papr has sufficiently resolved most of the identified issues. 
```

Comment:
"most of the identified issues" -- maybe we include something like "other identified issues were disputed and not found worthy of fixes" or something?